src/asm.c

```diff
@@ -78,7 +78,11 @@ int main(int argc, char *argv[]) {
 	}
 	fclose(file_tmp); /* close temporary file */
 	remove(tmpdev);   /* delete temporary file */
-	return (1);       /* close files, release memory, exit */
+
+	/* Originally 'main()' returned (1), which by VMS convention is a success
+	   code. Changed to (0) to match the UNIX convention. */
+	
+	return (0);       /* close files, release memory, exit */
```

Comment:
I wasn't sure what your preference was regarding comments about historical changes.

I'm happy to remove the comment and/or note the change in the README.md instead.